clients/ultralight/ultralight.sh

```diff
@@ -7,13 +7,13 @@ IP_ADDR=$(hostname -i | awk '{print $1}')
 FLAGS=""
 
 if [ "$HIVE_CLIENT_PRIVATE_KEY" != "" ]; then
-    FLAGS="$FLAGS --pk=0x1a2408021220$HIVE_CLIENT_PRIVATE_KEY"
+    FLAGS+=" --pk=0x1a2408021220$HIVE_CLIENT_PRIVATE_KEY"
 fi
 
 if [ "$HIVE_PORTAL_NETWORKS_SELECTED" != "" ]; then
-    FLAGS="$FLAGS --networks=$HIVE_PORTAL_NETWORKS_SELECTED"
+    FLAGS+=" --networks=$HIVE_PORTAL_NETWORKS_SELECTED"
 else
-    FLAGS="$FLAGS --networks=history"
+    FLAGS+=" --networks=history"
```

Comment:
```suggestion
    FLAGS="$FLAGS --networks=history"
```
missed 3 in this file